static/app/components/projects/appStoreConnectContext/utils.tsx

```diff
@@ -18,27 +18,11 @@ export const appStoreConnectAlertMessage = {
 export function getAppConnectStoreUpdateAlertMessage(
   appConnectValidationData: AppStoreConnectValidationData
 ) {
-  if (appConnectValidationData.itunesSessionValid === false) {
+  if (appConnectValidationData.promptItunesSession === true) {
```

Comment:
in case this property is always a boolean we can change the code to be like:


```suggestion
  if (appConnectValidationData.promptItunesSession) {
```